geoportal/c2cgeoportal_geoportal/lib/caching.py

```diff
@@ -123,72 +124,52 @@ def invalidate_region(region: Optional[str] = None) -> None:
         get_region(region).invalidate()
 
 
-class HybridRedisBackend(RedisBackend):  # type: ignore
+class HybridRedisBackend(CacheBackend):  # type: ignore
     """A memory and redis backend."""
```

Comment:
Improve the description Like
A Dogpile cache backend with a memory cache backend in front of a Redis backend for performance.